app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: str) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
```

Comment:
You should access class attribute through class name not `self`

Reply:
Fixed